darwin/importer/importer.py

```diff
@@ -40,6 +39,13 @@
 from darwin.utils import flatten_list, secure_continue_request
 from darwin.version import __version__
 
+try:
+    from mpire import WorkerPool
+
+    MPIRE_AVAILABLE = True
+except ImportError:
+    MPIRE_AVAILABLE = False
+
```

Comment:
This allows users to solve any issues with `mpire` just by removing the package locally.  In edge cases, users can just `pip remove mpire` or equivalent, and the system will stop trying to use it.